about.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About | Dev Conference 2023</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+    <link rel="stylesheet" href="./assests/css/styles.css">
+</head>
+
+<body>
+    <header>
+        <div class="headline">
+            <div class="container">
+                <div class="d-flex ms-auto">
+                    <span>
+                        <i class="fa-brands fa-facebook-f"></i>
+                    </span>
+                    <span>
+                        <i class="fa-brands fa-twitter"></i>
+                    </span>
+                    <span>English</span>
+                    <span>My page</span>
+                    <span>Logout</span>
+                </div>
+            </div>
+
+        </div>
+        <div class="bg-white navigation">
+            <div class="container">
+                <div>
+                    <nav class="navbar navbar-expand-lg bg-white">
+                        <div class="container-fluid">
+                            <a class="navbar-brand d-none d-md-flex" href="index.html">
+                                <img src="./assests/img/devcon5.webp" width="100" alt="DEV CONN" />
+                            </a>
+                            <button class="navbar-toggler collapsed" type="button" data-bs-toggle="collapse"
+                                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+                                aria-expanded="false" aria-label="Toggle navigation">
+                                <span class="navbar-toggler-icon"></span>
+                                <span class="navbar-close-icon"><i class="fa-lg fa-solid fa-times"></i></span>
+                            </button>
+                            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                                <ul class="navbar-nav ms-md-auto mb-2 mb-lg-0">
+                                    <li class="nav-item">
+                                        <a class="nav-link active" aria-current="page" href="/index.html">Home</a>
+                                    </li>
```

Comment:
- [ ] Currently, your home menu link is not working. Please make sure you added the correct `href` attribute data. (tip: take a look at the mobile menu you used to redirect the user to the about page).